.github/CONTRIBUTING.md

```diff
@@ -79,21 +53,20 @@ The form lists all employees who are authorized to contribute on behalf of your
+All components should have associated unit tests created with a code coverage score of at least 85%. Be an overachiever and shoot for 100% :) 
+Components also have visual regression tests enabled. [Learn how to add to our testing suite](https://github.com/SAP/fundamental-react/wiki/Testing)
 
 ### How to contribute - the Process
 
 1.  Make sure the issue you've filed in the [issue tracker] has the label "contribution welcome" - otherwise, it is not ready to be worked on.
-2.  Fork the Fundamental-react repository to your GitHub account.
-3.  Create a branch for your issue or feature, and commit or push your changes on that branch.
-4.  Create a Pull Request from your forked repository to github.com/SAP/fundamental-react. In the subject of the pull request, use "fix:" to denote a bug fix, "feat:" to denote an enhancement or "chore:" for small configuration updates and briefly describe the bug fix or enhancement you're contributing. In the pull request description, please provide a link to the issue in the issue tracker. **Do not include "BREAKING CHANGE" in the subject or description.**
-5.  Follow the link posted by the CLA assistant to your pull request and accept it, as described above.
-6.  Wait for our code review and approval. We may ask you for additional commits, or make changes to your pull request ourselves.
+1.  Fork the Fundamental-react repository to your GitHub account.
+1.  Create a branch for your issue or feature, and commit or push your changes on that branch.
+1.  Create a Pull Request from your forked repository to github.com/SAP/fundamental-react. In the subject of the pull request, use "fix:" to denote a bug fix, "feat:" to denote an enhancement or "chore:" for small configuration updates and briefly describe the bug fix or enhancement you're contributing. In the pull request description, please provide a link to the issue in the issue tracker. **Do not include "BREAKING CHANGE" in the subject or description.**
+1.  Follow the link posted by the CLA assistant to your pull request and accept it, as described above.
+1.  Wait for our code review and approval. We may ask you for additional commits, or make changes to your pull request ourselves.
     - Note that the Fundamental-react developers also have their regular duties so, depending on the required effort for reviewing, testing, and clarification, this may take a while.
-7.  Once the change has been approved, we inform you in a comment.
-8.  Your pull request cannot be merged directly into the branch (internal SAP processes), but is merged internally and immediately appears in the public repository as well.
-9.  We close the pull request. You can then delete the now obsolete branch.
+1.  Once the change has been approved, an admin will merge the pull request into master on your behalf. You can then delete the now obsolete branch.
```

Comment:
numbering goofed up?

Reply:
nah I just wanted to use 1. 1. 1. so that markdown does the work if we want to add new items without re-doing the numbering - future proof for laziness :)